<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyController</class>
 <widget class="QWidget" name="MyController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>341</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>170</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>   Окружность(Эллипс)                    Спектр </string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>х</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rx</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="xEdit">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>40</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rxEdit">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="yEdit">
       <property name="geometry">
        <rect>
         <x>123</x>
         <y>40</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>213</x>
         <y>77</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ΔR_x</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>97</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ryEdit">
       <property name="geometry">
        <rect>
         <x>124</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>20</width>
         <height>161</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="drEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>77</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="drawellipseButton">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>130</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Нарисовать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="drawSunButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Нарисовать</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>90</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ry</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rxstartEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>28</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>219</x>
         <y>103</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>N</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>219</x>
         <y>29</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rx</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>103</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>219</x>
         <y>52</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ry</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rystartEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>53</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Параметры отрисовщика</string>
      </property>
      <widget class="QLabel" name="linelabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>26</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Размер пикселя</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>53</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет отрезка</string>
       </property>
      </widget>
      <widget class="QLabel" name="fonlabel">
       <property name="geometry">
        <rect>
         <x>199</x>
         <y>80</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>54</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sizeEdit">
       <property name="geometry">
        <rect>
         <x>167</x>
         <y>26</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="foncolorButton">
       <property name="geometry">
        <rect>
         <x>229</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="linecolorButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="fonButton">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>107</y>
         <width>100</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sizePixelButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>25</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Выбор алгоритма</string>
      </property>
      <widget class="QComboBox" name="algoComboBox">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>30</y>
         <width>331</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Брезенхем</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Средней точки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>С исп. канонического уравнения</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>С исп. параметрического уравнения</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Стандартный</string>
        </property>
       </item>
      </widget>
      <widget class="QRadioButton" name="circleButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Окружность</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="ellipseButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>100</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Эллипс</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
